_posts/2022-03-16-solidity-0.8.13-release-announcement.md

```diff
@@ -0,0 +1,235 @@
+directive and implements "go to definition" for the language server.
+
+Furthermore, compiling via the new Yul IR pipeline is now considered production ready.
+
+
+## Important Bugs
+
+When ``abi.encodeCall`` was introduced in Solidity 0.8.11,
+hex literals (``0x1234``) and string literals (``"abcd"``) were not handled properly.
+Please read more about it in the [security alert](https://blog.soliditylang.org/2022/03/16/encodecall-bug/).
+
+## Notable New Features
+
+### Yul IR Pipeline Production Ready
+
+We have been working on Yul as an intermediate language for Solidity for several years now.
+Yul in itself has been considered production ready for a while - at least since
+ABI coder V2 was marked "stable", because it makes heavy use of Yul. The other part,
+code generation from Solidity to Yul, can now also be considered stable and production ready.
+
+It has been feature complete for several months now. Since the beginning, we always ran all our
+tests through both compiler pipelines. Over the previous weeks, we added several more complex
+real-world contracts to our testing infrastructure and fixed some minor bugs in the component.
+The new code generator has been very stable for a long time now. The main reason for not
+marking it "non-experimental" earlier was that it still had problems with limited stack space,
+which we were able to fix now.
+
+You can enable the new pipeline using the ``--via-ir`` commandline switch or by setting
+``settings.viaIR`` to ``true`` in Standard JSON input (i.e. at the same level where
+you would have the ``optimizer`` key).
+
+It is not recommended to use the Yul compiler pipeline without the optimizer because the unoptimized output
+is primarily meant to be a straightforward transformation of the high-level Solidity code
+and it has many additional checks compared to the old compiler pipeline.
+This makes it more
+modular and easier to audit while the new Yul-based optimizer is powerful enough to remove redundancies
+from the optimized version.
+
+The performance of the new pipeline is not yet always superior to the old one, but it
+can do much higher-level optimization across functions, so please try it out and give us feedback!
+
+Finally, in some edge cases, code compiled via the new compiler pipeline
+behaves different than code compiled via the old compiler pipeline.
+The most prominent is the initialization order of state variables that depend
+on the result of constructors in other contracts.
+Others are related to weird uses of modifiers, dependence of evaluation order inside
+statements or dirty data. You can find a list of all the changes
+in our [documentation](https://docs.soliditylang.org/en/develop/ir-breaking-changes.html).
+
+### Memory-Safe Assembly / Stack Too Deep
+
+One important feature of the new compiler pipeline is that it can move stack variables to memory
+and thus avoid the "stack too deep" issue. In order to do this safely, the compiler needs to know
```

Comment:
```suggestion
and thus avoid the "stack too deep" issue in a lot of cases. In order to do this safely, the compiler needs to know
```
to be a bit more conversative :-)... we probably don't need to mention that it won't happen in recursive functions, but we should mention that it's only done by the optimizer.